NuGet.config

```diff
@@ -16,7 +16,9 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsPackages" value="..\bin\Debug\NuGet" />
```

Comment:
The solution should have a parent folder with all the child projects in it, that's how it is currently setup.